draft-ietf-mls-protocol.md

```diff
@@ -1639,30 +1640,13 @@ the group.
 It is RECOMMENDED for the application generating exported values
 to refresh those values after a Commit is processed.
 
-## Recovery Keys
```

Comment:
Why remove this section? If I understand correctly, it fits well into the subsections explaining the use of the various keys derived from the key schedule.

Reply:
You did not define how to do recovery. Define how to do recovery in another PR and you can add it back

Reply:
This Section is not about keys for recovery specifically, but about the `recovery_secret`, which is the basis for internal PSK derivation. I agree that the name is not ideal, though.